server/verifiers/twitter-verifier/verifier/src/main/java/org/kaaproject/kaa/server/verifiers/twitter/verifier/TwitterUserVerifier.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2015 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaaproject.kaa.server.verifiers.twitter.verifier;
+
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.apache.http.message.BasicHttpEntityEnclosingRequest;
+import org.kaaproject.kaa.server.common.verifier.AbstractKaaUserVerifier;
+import org.kaaproject.kaa.server.common.verifier.UserVerifierCallback;
+import org.kaaproject.kaa.server.common.verifier.UserVerifierContext;
+import org.kaaproject.kaa.server.verifiers.twitter.config.gen.TwitterAvroConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.InvalidKeyException;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.*;
+
+public class TwitterUserVerifier extends AbstractKaaUserVerifier<TwitterAvroConfig> {
+    private static final Logger LOG = LoggerFactory.getLogger(TwitterUserVerifier.class);
+    private static final String TWITTER_URL = "https://api.twitter.com/1.1/account/verify_credentials.json";
```

Comment:
Its looks like url include version, I think is better to move this to configuration